extensions/docker/src/docker-cli.ts

```diff
@@ -0,0 +1,133 @@
+/**********************************************************************
+ * Copyright (C) 2022 Red Hat, Inc.
```

Comment:
```suggestion
 * Copyright (C) 2023 Red Hat, Inc.
```

Reply:
it's marked as resolved but it's still displaying 2022 ?
if it's a copy it should be `2022-2023` or `2023`

Reply:
Done